DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -418,5 +418,15 @@ public void ActivateDelegateUser_activates_delegate_user()
             // Then
             userDataService.GetDelegateUserById(delegateUserId)!.Active.Should().BeTrue();
         }
+
+        [Test]
+        public void AddDelegateToGroup_get_delegates_not_registered_for_group_by_groupId()

```

Comment:
1. The test name prefix does not match the SIT.
2. Also I think this title could be sligtly more descriptive in what it's aims are such as: 
 ` GetDelegatesNotRegisteredForGroupByGroupId_returns_expected_number_of_delegates(...)`

Reply:
I have renamed it as suggested